app/models/space/serializer.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+# Serializes {Space}s for programmatic consumption
+class Space::Serializer < ApplicationSerializer
+  # @return [Space]
+  alias space resource
+
+  def to_json(*_args)
```

Comment:
Feels like it is time to start introducing https://github.com/jsonapi-serializer/jsonapi-serializer (or similar), making this all a little better. Maybe in a follow-up PR.

Reply:
Concur.